python/samples/04.ai.a.teamsChefBot/src/config.py

```diff
@@ -0,0 +1,23 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
TTK only generates the `.env` file, it doesn't load the envs to the process.
The VS Code Python extension does the trick in the samples. If you just run `python app.py`, the app will not able to find the envs.

I personally prefer to clearly express the env loading logic in source code, so there won't be hidden logic in the sample. And this may also be helpful when user moves the app to the cloud, where no VS Code Python extension exists.